### ISSUE TITLE
[Feature] support jedis-5.x

### ISSUE THREAD

**Issue by luger1990** [apache/skywalking#11747]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

The current skywalking java agent([apache-skywalking-java-agent-9.1.0.tgz](https://dlcdn.apache.org/skywalking/java-agent/9.1.0/apache-skywalking-java-agent-9.1.0.tgz)) dosen't support jedis 5.x

### Are you willing to submit a pull request to implement this on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
We are open to new plugin contributions. If you want to submit a new one, welcome. 
But for issue tracker, unless you are going to submit one in short term, we wouldn't keep this one open.
Too many kind of requests could be opened. It is hard to follow.

**Comment by luger1990** [comment]
> We are open to new plugin contributions. If you want to submit a new one, welcome. But for issue tracker, unless you are going to submit one in short term, we wouldn't keep this one open. Too many kind of requests could be opened. It is hard to follow.

Do your team have any time plan for release the new plugin? Thanks

**Comment by wu-sheng** [comment]
What do you mean my team? In a community, there is nothing about my team. SkyWalking doesn't work in this way.

**Comment by luger1990** [comment]
> What do you mean my team? In a community, there is nothing about my team. SkyWalking doesn't work in this way.

I want to know when will support jedis-5.x, i will roll back to use jedis-4.x if it take a long time.

**Comment by wu-sheng** [comment]
No one is working on that. That is only thing I can say. No timeline, maybe tomorrow submit a pull request, maybe 1-2 years. No one knows.

Like I said, this is not how SkyWalking works. This is not a work for anyone, including me.

**Comment by wu-sheng** [comment]
Jedis 5.x support has been added in apache/skywalking-java#803.

The Jedis 4.x plugin is fully API-compatible with Jedis 5.x — all intercepted classes (`Connection`, `Pipeline`, `Transaction`, `ConnectionProvider`) and methods are unchanged. The plugin was silently not activating on 5.x because the witness method `Pipeline.persist(1 arg)` moved from `Pipeline` to parent `PipeliningBase` in 5.x.

Fix: changed the witness to `Connection.executeCommand(1 arg)` which exists in both 4.x and 5.x but not in 3.x.

Locally verified: Jedis 5.2.0 passed all test cases.
